src/components/organisms/AdminVenues/AdminVenues.scss

```
@@ -4,10 +4,15 @@
   width: 100%;
   height: 100%;
 
-  &__create-button {
+  &__header {
     display: flex;
     align-items: center;
     justify-content: space-between;
+
+    &--title {
+      font-size: 2em;
+      font-weight: bolder;
+    }
   }
```

The title isn't a 'modifier', it's just more just another 'element':

- http://getbem.com/naming/

```suggestion
  &__header {
    display: flex;
    align-items: center;
    justify-content: space-between;
  }
  
  &__title {
    font-size: 2em;
    font-weight: bolder;
  }
```